sdk/keyvault/azkeys/client.go

```diff
@@ -136,9 +136,9 @@ func createKeyResponseFromGenerated(g generated.KeyVaultClientCreateKeyResponse)
 	}
 }
 
-// CreateKey - The create key operation can be used to create any key type in Azure Key Vault.
-// If the named key already exists, Azure Key Vault creates
-// a new version of the key. It requires the keys/create  permission.
+// CreateKey can be used to create any key type in Azure Key Vault.  If the named key already exists,
+// Azure Key Vault creates a new version of the key. It requires the keys/create permission. Pass in
```

Comment:
How about just: "Default options are used if nil is passed."

Reply:
Or "Pass nil to use the default options"